cmd/sealer/cmd/cluster/apply.go

```diff
@@ -132,6 +132,9 @@ func NewApplyCmd() *cobra.Command {
 			if len(md) > 0 || len(nd) > 0 {
 				logrus.Warnf("scale down not supported: %v, %v, skip them", md, nd)
 			}
+			if len(md) > 0 {
+				return fmt.Errorf("please ensure all master ips info exist in your clusterfile: %s", applyFlags.ClusterFile)
```

Comment:
Now that it is an apparent error, why not directly illustrate the error case in the forbidden way, rather than still let users ensure, check and blablabla. 

"严禁随地大小便" 即可，不建议使用“请确认是否有随地大小便”.